templates/webapi/admin/job_template/index.html.ep

```diff
@@ -103,7 +103,7 @@
         <form action="#" id="editor-form" class="form-horizontal" style="display: none;"
               data-put-url="<%= url_for('apiv1_job_templates_schedules' => (id => $group->id)) %>" data-reference="<%= $yaml_template %>">
             <div class="form-group row">
-                <label for="editor-template" class="col-sm-5 control-label editor-yaml-guide" style="display: none;">
+                <label for="editor-template" class="col-sm-5 control-label editor-yaml-guide" style="display: none; text-align: left;">
```

Comment:
I don't know why we have "display: none" in the html template and not in CSS but I think the text-align should go into CSS.

@Martchus ?

Reply:
We have `display: none;` there because the element is dynamically hidden and shown and nobody bothered making an extra class just for one CSS property. This `text-align: left` would fit better into the CSS file, especially because the element has already the class `.editor-yaml-guide` so we should just use that.

Reply:
If I create a css for `editor-yaml-guide`, I guess I can also move `display: none` there.
I don't see a reason why this should stay in the HTML then.

Reply:
I moved both settings to `assets/stylesheets/forms.scss`